configs/coins/bsc.json

```diff
@@ -0,0 +1,70 @@
+{
+  "coin": {
+    "name": "BSC Smart Chain",
+    "shortcut": "BNB",
+    "label": "BSC Smart Chain",
+    "alias": "bsc"
+  },
+  "ports": {
+    "backend_rpc": 8203,
+    "backend_p2p": 38403,
+    "backend_http": 8303,
+    "blockbook_internal": 9203,
+    "blockbook_public": 9303
+  },
+  "ipc": {
+    "rpc_url_template": "ws://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_timeout": 25
+  },
+  "backend": {
+    "package_name": "backend-bsc",
+    "package_revision": "satoshilabs-1",
+    "system_user": "bsc",
+    "version": "1.1.18-hf",
+    "binary_url": "https://github.com/bnb-chain/bsc/releases/download/v1.1.18_hf/geth_linux",
+    "verification_type": "sha256",
+    "verification_source": "1427ef4bc19e51fe9dead2d211ff6ebebe610cbbe7a0e1b35870f01cfed6473e",
+    "extract_command": "mv ${ARCHIVE} backend/geth && chmod +x backend/geth && echo",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '[[ ! -d {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/geth/chaindata ]] && {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth init --datadir {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/genesis.json 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log; {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth --networkid 56 --syncmode full --txlookuplimit 0 --ipcdisable --cache 8000 --nat none --rpc.allow-unprotected-txs --datadir {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend --port {{.Ports.BackendP2P}} --ws --ws.addr 127.0.0.1 --ws.port {{.Ports.BackendRPC}} --ws.origins \"*\" --ws.api \"eth,net,web3,debug,txpool\" --http --http.port {{.Ports.BackendHttp}} --http.addr 127.0.0.1 --http.corsdomain \"*\" --http.vhosts \"*\" --http.api \"eth,net,web3,debug,txpool\" 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
```

Comment:
can leverage `exec_script` as well once optimism is in or cherry-pick to make this command less crazy